stock_planning_procurement_generated_by_plan/models/stock_planning.py

```diff
@@ -13,27 +13,52 @@ class StockPlanning(models.Model):
     def _get_to_date(self):
         super(StockPlanning, self)._get_to_date()
         proc_obj = self.env['procurement.order']
+        move_obj = self.env['stock.move']
+        moves = move_obj._find_moves_from_stock_planning(
+            self.company, self.scheduled_date, product=self.product,
+            location_id=self.location, location_dest_id=self.env.ref(
+                'stock_reserve.stock_location_reservation'))
+        self.outgoing_to_date_reserve_destination = sum(
+            moves.mapped('product_uom_qty'))
         states = ('confirmed', 'exception')
         procurements = proc_obj._find_procurements_from_stock_planning(
-            self.company, self.scheduled_date, states, product=self.product,
-            warehouse=self.warehouse, location_id=self.location,
-            without_purchases=True, without_productions=True, level=0)
+            self.company, self.scheduled_date, states=states,
+            product=self.product, location_id=self.location,
+            without_purchases=True, without_productions=True, level=0,
+            from_plan=False)
         self.procurement_incoming_to_date = sum(
             procurements.mapped('product_qty'))
         procurements = proc_obj._find_procurements_from_stock_planning(
-            self.company, self.scheduled_date, states, product=self.product,
-            warehouse=self.warehouse, location_id=self.location,
-            without_purchases=True, without_productions=True, level=1)
-        self.procurement_plan_incoming_to_date = sum(
+            self.company, self.scheduled_date, states=states,
+            product=self.product, location_id=self.location,
+            without_purchases=True, without_productions=True, level=0,
+            from_plan=True)
+        self.procurement_plan_incoming_to_date_level0 = sum(
+            procurements.mapped('product_qty'))
+        procurements = proc_obj._find_procurements_from_stock_planning(
+            self.company, self.scheduled_date, states=states,
+            product=self.product, location_id=self.location,
+            without_purchases=True, without_productions=True, level=1,
```

Comment:
level=1 sería incorrecto. Debería ser level>0 and level <100 para coger los procurements generados en el despliegue de niveles del plan